x-pack/plugins/endpoint/public/applications/endpoint/view/policy/policy_details.tsx

```diff
@@ -30,7 +30,7 @@ import {
   apiError,
 } from '../../store/policy_details/selectors';
 import { WindowsEventing } from './policy_forms/eventing/windows';
-import { PageView, PageViewHeaderTitle } from '../../components/page_view';
+import { PageView, PageViewHeaderTitle } from '../components/page_view';
```

Comment:
@parkiino conflict was here. I think you removed `WindowsEventing` in master but not in 7.x

Reply:
oh i still need to do my backport osrry